src/views/product-tutorials-view/components/product-view-content/components/logo-card-list/index.tsx

```diff
@@ -1,13 +1,30 @@
-import s from './logo-card-list.module.css'
+import CardsGrid from 'components/cards-grid-list'
+import CollectionCard, {
+  CollectionCardPropsWithId,
+} from 'components/collection-card'
+import { formatCollectionCard } from 'components/collection-card/helpers'
 import { LogoCardListProps, LogoCardListItem } from './types'
 
 function LogoCardList({ items }: LogoCardListProps): JSX.Element {
```

Comment:
This component could do with a rename (it's really collection cards in disguise), but have kept the name as-is for now to keep the link with the `block` type in source content more clear 👍